### PR TITLE
Closes #115 package maintainer update

### LINKED ISSUE #115: DESCRIPTION file updates for Authors / Package maintainer
### Background Information

New authors and new package maintainer to add/update

### Definition of Done

- [ ] Add Lina Patil and Fanny Gautier (incl. ORCID = "0009-0004-3581-0131") as authors.
- [ ] Update Package maintainer with Fanny's info, moving Edoardo as Author
- [ ] Sort out the names by alphabetical order (package maintainer remaining on top of the list)

### PR DESCRIPTION
**Pull request by manciniedoardo**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build pharmaverseadam site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new datasets occur on the "[Reference](https://pharmaverse.github.io/pharmaverseadam/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -3,15 +3,17 @@ Package: pharmaverseadam
...
+    person("Fanny", "Gautier", email = "fanny.gautier@cytel.com", role = c("aut", "cre"),
+           comment = c(ORCID = "0009-0004-3581-0131")),
+    person("Stefan", "Bundfuss", role = "aut",
+           comment = c(ORCID = "0009-0005-0027-1198")),
     person("Stefan", "Bundfuss", role = "aut",
            comment = c(ORCID = "0009-0005-0027-1198")),
+    person("Gayatri", "G", role = "aut"),      
+    person("Edoardo", "Mancini", email = "edoardo.mancini@roche.com", role = "aut",
+           comment = c(ORCID = "0009-0006-4899-8641")),
     person("Gayatri", "G", role = "aut"),
```
Comment (Lina2689) [inline DESCRIPTION:15]: "Gayatri G" and "Stefan Bundfuss" mentioned twice. Please update and sort out the names by alphabetical order

File: NEWS.md [hunk 2]
```diff
@@ -6,6 +6,8 @@
 
 ## Documentation
 
+- The package maintainer was updated from Edoardo Mancini to Fanny Gautier. (#186)
+
```
Comment (Lina2689) [inline NEWS.md:10]: issue number should be #115 instead of #186

File: DESCRIPTION [hunk 3]
```diff
@@ -3,15 +3,13 @@ Package: pharmaverseadam
...
+           comment = c(ORCID = "0009-0004-3581-0131")),
     person("Stefan", "Bundfuss", role = "aut",
            comment = c(ORCID = "0009-0005-0027-1198")),
-    person("Gayatri", "G", role = "aut"),
-    person("Daphne", "Grassely", role = "aut"),
-    person("Zelos", "Zhu", role = "aut"),
-    person("Sadchla", "Mascary", role = "aut"),
+    person("Edoardo", "Mancini", email = "edoardo.mancini@roche.com", role = "aut",
+           comment = c(ORCID = "0009-0006-4899-8641")),
+      person("Lina", "Patil", email = "lina.patil@cytel.com", role = "aut"),
```
Comment (Lina2689) [inline DESCRIPTION:12]:
```suggestion
 person("Lina", "Patil", email = "lina.patil@cytel.com", role = "aut"),
```

File: README.md [hunk 4]
```diff
@@ -75,3 +75,9 @@ Secondly, the user runs `data-raw/create_adams_data.R`. This script handles the
    - Variable-level documentation (names and labels),
    - Source information indicating which template and package generated it,
    - Example usage with `data("<dataset>")`.
+   
+## Acknowledgments
+
+Along with the authors and contributors, thanks to the following people for their work on the package:
+
+Daphne Grassely, Sadchla Mascary, Kangjie Zhang and Zelos Zhu.
```
[the remark this answers is not in the document]
Comment (manciniedoardo) [reply README.md]: no, thanks for spotting, added

**Review by Lina2689 — APPROVED** [review]
Thanks for the updates! It's good now.
